docs/mobile/mobile_overview.md

```diff
@@ -128,7 +124,7 @@ You can contribute in different ways:
 
 - Theming: You can optimize the current CSS files or even create your own theme.
 
-- Widgets: Widget / Feature missing? Create an widget an post it back to us.
+- Widgets: Widget / Feature missing? Create a widget a post it back to us.
```

Comment:
s/Create a widget a post/Create a widget and submit it as a Pull Request/

(I think that's what's intended here...???)